### DISCUSSION TITLE
USB High speed capable devices

### DISCUSSION THREAD

**Discussion by perigoso** [hathach/tinyusb#779] [Q&A] [answered]
Looking at the readme only some of the stm32 family clearly supports high speed.

I need a usb 2.0 high speed device for a project, what supported devices can i use?

~~I was looking into using the ATMSAMS70 ([datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/SAM-E70-S70-V70-V71-Family-Data-Sheet-DS60001527D.pdf)) device, but Its not in the supported list, though some of the same family are, like the ATSAME5x, but these don't support high speed, what is the likelihood that the existing driver works for the SAMS70?~~

Edit: I just saw this #693, I think this answers my second question.

**Comment by hathach** [comment, marked as answer]
NXP has lots of HS USB such as lpc18, 43, 55 (not yet supported), all imxrt mcus. You can search for `TUSB_SPEED_HIGH`, those that reports that in reset is capable of HS

**Reply by perigoso** [reply]
I'm going to use the ATSAMS70, thank you for the suggestions.

I think it could be interesting to have more details about supported devices, for instance maybe a table with things like High speed support host support etc

**Reply by hathach** [reply]
table of features is nice indeed, I will try to do it later on
